configure

```diff
@@ -1,26 +1,36 @@
 #!/bin/sh
 
 # Use pkg-config or xml2-config to get the include directories for libxml-2.0
-xml2_include_paths=""
-if which xml2-config >/dev/null 2>&1; then
-    xml2_include_paths=$(xml2-config --cflags)
-elif which pkg-config >/dev/null 2>&1; then
-    xml2_include_paths=$(pkg-config --cflags libxml-2.0)
+libxml2_cflags=""
+if command -v xml2-config >/dev/null 2>&1; then
+    libxml2_cflags=$(xml2-config --cflags)
+    libxml2_libs=$(xml2-config --libs)
+elif command -v pkg-config >/dev/null 2>&1; then
+    libxml2_cflags=$(pkg-config --cflags libxml-2.0)
+    libxml2_libs=$(pkg-config --libs --static libxml-2.0)
 else
     echo "Error: Neither xml2-config nor pkg-config is available"
     exit 1
 fi
 
-if [ -n "$xml2_include_paths" ]; then
-    PKG_CFLAGS="$xml2_include_paths"
-    echo "libxml2 include directories: $xml2_include_paths"
+if [ -n "$libxml2_cflags" ]; then
+    PKG_CFLAGS="$libxml2_cflags"
+    echo "libxml2 include directories: $libxml2_cflags"
 else
     echo "Error: libxml2 include directory not found"
     exit 1
 fi
 
+if [ -n "$libxml2_libs" ]; then
+    PKG_LIBS="$libxml2_libs"
+    echo "libxml2 library link flags: $libxml2_libs"
+else
+    echo "Error: libxml2 library link flags not found"
+    exit 1
+fi
+
 echo "# Generated from Makevars.in, do not edit by hand" > src/Makevars.new
-sed -e "s|@cflags@|$PKG_CFLAGS|" src/Makevars.in >> src/Makevars.new
+sed -e "s|@cflags@|$PKG_CFLAGS|" -e "s|@libs@|$PKG_LIBS|" src/Makevars.in >> src/Makevars.new
```

Comment:
These placeholders are in `Makevars.in` .

Reply:
Forgot to commit that change.